reference/requirements.rst

```diff
@@ -28,8 +28,8 @@ Required
 
 .. caution::
 
-    Be aware that Symfony has some known limitations when using PHP 5.3.16.
-    For more information see the `Requirements section of the README`_.
+    Be aware that Symfony 5.3.16 is not suitable to run Symfony,
```

Comment:
Thanks for spotting this @viviengaetan! I'm going to fix the typo. :sweat_smile: